lib/exercism/github_source.rb

```diff
@@ -1,25 +1,40 @@
 class GithubSource
-  attr_reader :github, :user, :slug, :code
+  attr_reader :user, :slug, :code, :submission
 
   def initialize(submission)
-    @github = Github.new
+    @submission = submission
     @user = submission.user
     @slug = submission.slug
     @code = submission.code
   end
 
   def solution
-    trees = github.git_data.trees.get user.username, slug, code
+    Octokit.configure do |c|
+      c.login = 'SaiPramati'
+      c.password = 'pramati123'
+    end
+    git_tree_source = Octokit.tree("#{submission.user.username}/#{submission.slug}",
+                                    submission.solution.values.first, recursive: true)
```

Comment:
Line is too long. [86/80]<br>Align the parameters of a method call if they span more than one line.